### DISCUSSION TITLE
Aquila X2 with Alex’s firmware version 1.3.4.1 issues

### DISCUSSION THREAD

**Discussion by Olly1965** [Jyers/Marlin#1520] [General Discussion]
Hi,
First of all great job on this firmware….
For some reason I could not get version 1.3.5 to work on my Aquila X2, so went for the one below…
So my first question, if and when I do upgrade this to the newer version 1.3.5, will I need to flash the screen again..?

Secondly the version I am using does not seem to work properly with the filament runout sensor on the Aquila X2 is this a known issue or not…and is it fixed in the 1.3.5 version…it throws a breakage error, when there is no breakage….
Cheers
Olly

**Comment by darco415** [comment]
Who's Alex?

**Reply by Olly1965** [reply]
Alexqzd who created the firmware for this printer….
